packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -225,6 +233,36 @@ export function TransferPanel() {
     connectionState
   })
 
+  const isTokenAlreadyImported = useMemo(() => {
```

Comment:
Checks if token is already imported before we show the token import dialog. This way when we refresh the page we won't show the dialog all the time.